src/android/CordovaADAL/src/com/microsoft/aad/adal/CordovaAdalPlugin.java

```diff
@@ -122,12 +122,14 @@ private boolean acquireTokenAsync(String authority, String resourceUrl, String c
         }
 
         if (userId != null) {
-            ArrayList<TokenCacheItem> tokensForUserId =
-                    ((DefaultTokenCacheStore)authContext.getCache()).getTokensForUser(userId);
-
-            if (tokensForUserId.size() > 0) {
-                // Try to acquire alias for specified userId
-                userId = tokensForUserId.get(0).getUserInfo().getDisplayableId();
+            ITokenCacheStore cache = authContext.getCache();
+            if (cache instanceof ITokenStoreQuery) {
```

Comment:
Yes. Only `DefaultTokenCacheStore` implements this interface. In case if Company portal is installed, context returns instance of 'ITokenCacheStore' interface.